src/test/scala-2.11/markets/clearing/engines/CDAMatchingEngineSpec.scala

```diff
@@ -321,12 +320,12 @@ class CDAMatchingEngineSpec extends TestKit(ActorSystem("CDAMatchingEngineSpec")
 
       Then("the matching engine should generate a PartialMatch")
 
-      val fill = PartialMatch(bidOrder, askOrder, bidPrice)
+      val residualAskQuantity = askQuantity - bidQuantity
+      val (filledAskOrder, residualAskOrder) = askOrder.split(residualAskQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1539841712/issues/source?bid=2921361&fileBranchId=3014278#l324)